prlworkflows/sqs.py

```diff
@@ -273,7 +276,6 @@ def reindex(self, new_indices):
 def enumerate_sqs(structure, subl_model, endmembers=True, scale_volume=True, skip_on_failure=False):
```

Comment:
No longer users the endmembers command, so this and the documentation should be removed.